[OSDOCS-9416]: Clarify Cluster Uninstallation warning when using AWS VPC Peering connections (#70838)
openshift/openshift-docs: modules/aws-vpc.adoc

[WARNING]
====

> Before attempting to uninstall a cluster, you must remove any VPC peering connections from the cluster's VPC.

Reviewer: Consider maybe (but it's your call): 

"Before you attempt to uninstall a cluster, you must remove any VPC peering connections from the cluster's VPC."

Reviewer 2: You could offer the user a little more context for why they need to take this action. https://redhat-documentation.github.io/supplementary-style-guide/#admonitions:~:text=disregard%20this%20information.-,WARNING,-Information%20about%20potential

> Before attempting to uninstall a cluster, you must remove any VPC peering connections from the cluster's VPC.

Reviewer 2: ```suggestion
You must remove any VPC peering connections from the cluster's VPC before attempting to uninstall a cluster. 
```

> Before you attempt to uninstall a cluster, you must remove any VPC peering connections from the cluster's VPC. Failure to do so might result in a cluster that is not completely uninstalled.

Reviewer 2: ```suggestion
Before you attempt to uninstall a cluster, you must remove any VPC peering connections from the cluster's VPC. Failure to do so might result in a cluster not completing the uninstall process.
```
